test/examples/profile-addEmailSubscription.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Vitalik",
+  "avatar": "ipfs://NnVTdgcFciJUoxqgsNe9Hy2R7rgvEb5jDu5Pa5mmEG3UghbubTf2M78jUXbKLqKM",
+  "about": "This is my about info",
+  "emailSubscription": {
+    "email": "test@email.com"
```

Comment:
We will hash this email while saving to IPFS? i think users may not be able to verify their signature using tools like https://signator.io/

Not sure what's the alternative here

Reply:
In that case, we could just remove email from the object before saving to ipfs

Reply:
but that will still show it as wrong signature, as we remove one field from signed message

Reply:
Will move out the email subscription creation from profile